src/reactor.js

```diff
@@ -26,9 +27,13 @@ class Reactor {
   constructor(config = {}) {
     const debug = !!config.debug
     const baseOptions = debug ? DEBUG_OPTIONS : PROD_OPTIONS
+    // if defined, merge the custom implementation over the noop logger to avoid undefined lookups,
+    // otherwise, just use the built-in console group logger
+    const debugLogger = config.logging ? extend({}, NoopLogger, config.logging) : ConsoleGroupLogger
```

Comment:
Changed to `logger` for consistency with the rest of the options.  Let me know if you want it changed to first-cap.